es-design-system/pages/molecules/es-rating.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+    <div>
+        <es-rating
+            :read-only="false"
+            @change="changeEvent" />
+    </div>
+</template>
+<script>
+import { EsRating } from '@energysage/es-vue-base';
+
+export default {
+    name: 'EsRatingDocs',
+    components: {
+        EsRating,
+    },
+    methods: {
+        changeEvent($event) {
+            // eslint-disable-next-line no-alert
+            alert($event);
```

Comment:
This will end up being used in a review submission form. `@change` would be used to modify the forms data. `v-model` might work as well I didn't try it.